docs/upgrading-solidus/v4.1.mdx

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 1
+---
+
+import PRLink from '@site/src/theme/PRLink';
+import MinimalRequirements from '@site/src/theme/MinimalRequirements';
+
+# Solidus v4.1 (2023-06-27)
+
+<MinimalRequirements ruby="3.0" rails="7.0" />
+
+Solidus v4.1 is out! 🎉
+
+This release is relatively tiny and should be quite easy to upgrade. It starts to lay down the foundational
+work for the new Admin, which will be delivered gradually during the rest of the v4.x releases.
+
+In fact, with v4.1, stores can switch their Admin to the new color scheme, and allow operators to start
+getting used to it, reducing the impact of future changes.
+
+Let's see in detail the main things to be aware of when upgrading.
+
+:::info
+
+Please, when upgrading, remember to review our generic [upgrade guides](/getting-started/upgrading-solidus.mdx) and run:
+
+```
+bin/rails g solidus:update
+```
+
+You can also check the complete [Changelog](https://github.com/solidusio/solidus/blob/v4.1/CHANGELOG.md) in our repository.
+
+:::
+
+## <PRLink description="Add a new admin theme" number="5092" />
+
+https://github.com/solidusio/solidus/pull/5092 is the first step towards the new Admin. It introduces a new
+theme, which is a set of colors and styles that can be applied to the Admin.
+
+To switch to the new theme, you can set the following preference:
+
+```
+Spree::Backend::Config.theme = 'solidus_admin'
+```
+
+## <PRLink description="Allow changing the order recalculator" number="5110" />
+
+The `order.recalculate` and `order.recalculator` are introduced as aliases to the now deprecated
+`order.update` and `order.updater` in order to clarify the intention of these methods.
+
+This change also adds a preference to change the order recalculator (also known as order updater).
+The order recalculator, a very central piece of Solidus' infrastructure, carries a bit of technical debt.
+Unfortunately, it's extremely hard to change because every line of this class might have been overwritten
+by implementors.
```

Comment:
The tone here is a bit gloomy, but I guess anyone that ever worked with the _recalculator_ will know 😅 
(just a consideration, no change needed)

Reply:
You are right, I can omit that part: it is not relevant in the upgrade phase.

Reply:
Done!